test/controllers/resource_controller_test.rb

```diff
@@ -192,6 +192,51 @@ class ResourceControllerTest < ActionDispatch::IntegrationTest
     end
   end
```

Comment:
@alis-khadka could you add a test that verifies the-non dynamic case? Where we simply redirect to a CMS page 

See below for example: 
![Screenshot from 2022-07-10 14-01-15](https://user-images.githubusercontent.com/35935196/178156611-36aca726-cd48-44d6-b8b6-af0f3b8d100b.png)